lib/hdfsutils/utils/util.rb

```diff
@@ -36,5 +37,40 @@ def initialize(name, argv, optsproc = nil)
     def run
       fail 'Subclass of Util must override Util::run'
     end
+
+    #
+    # Useful operations common between utilities
+    #
+    def stat?(path)
```

Comment:
stat? is not a predicate (returns only true or false), so it should not end in a question mark.  Please rename stat?.